recipes/poco/all/conanfile.py

```diff
@@ -126,7 +132,7 @@ def requirements(self):
             self.requires("apr-util/1.6.1")
         if self.options.enable_netssl or self.options.enable_crypto or \
            self.options.get_safe("enable_jwt"):
-            self.requires("openssl/1.1.1n")
+            self.requires("openssl/1.1.1o")
```

Comment:
Should we upgrade sqlite3 too?

Reply:
I would rather not mix other upgrades into this pull-request. 
The update of openssl was mandated by the build system to actually build. An update of sqlite is not. 
If you need a newer version of sqlite in your project, you can always override the version in your project.

Reply:
please, update to the latest sqlite version too. It avoids other warnings when consuming Poco in Conan center and sqlite is outdated. There is no big deal updating dependencies in same PR, it's an usual action.

Reply:
I have updated sqlite and libmysqlclient as well. Updating libmysqlclient resolves a warning about different versions of openssl.